modules/openapi-generator/src/main/resources/swift4/AlamofireImplementations.mustache

```diff
@@ -64,6 +64,12 @@ open class AlamofireRequestBuilder<T>: RequestBuilder<T> {
         return Alamofire.SessionManager(configuration: configuration)
     }
 
+    open func createURLRequest() -> URLRequest? {
```

Comment:
@d-date what about adding a 1-liner explaining what this function does?

Reply:
@wing328  Thanks! Added